java/src/main/native/CMakeLists.txt

```diff
@@ -224,7 +224,7 @@ endif(USE_NVTX)
 ###################################################################################################
 # - link libraries --------------------------------------------------------------------------------
 
-target_link_libraries(cudfjni cudf rmm cudart cuda nvrtc)
+target_link_libraries(cudfjni cudf rmm cudart_static cuda nvrtc)
```

Comment:
Do we still want to statically link if the dependencies are not doing so?  Wondering if this should be conditional on the same static trigger in libcudf.

Reply:
The two cmake instances are not tied to each other in any way. We would have to do the same hacked up things we are doing in the pom.xml to try and detect what they were built with, which I really would rather avoid.  The only issue with static linking here is that the JNI library will be slightly larger.